test/truefi2/TrueCreditAgency.test.ts

```diff
@@ -385,6 +387,16 @@ describe('TrueCreditAgency', () => {
         .to.be.revertedWith('TrueCreditAgency: The pool is not whitelisted for borrowing')
     })
 
+    it('cannot borrow over the borrow limit', async () => {
+      await creditAgency.allowBorrower(borrower2.address, true)
+      await creditOracle.setScore(borrower2.address, 191)
+      await creditOracle.setMaxBorrowerLimit(borrower2.address, parseEth(100))
+
+      expect(await creditAgency.borrowLimit(tusdPool.address, borrower2.address)).to.eq(parseEth(80.51))
+      await expect(creditAgency.connect(borrower2).borrow(tusdPool.address, parseEth(80.51).add(1)))
```

Comment:
Would be nice to add a test case, where at first borrower borrows i.e. `parseEth(75)` and it works, and then attempts to borrow more - i.e. `parseEth(5.51).add(1)` and it fails.